src/specialOffer/SpecialOffer.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+
+import { Button, ButtonStatus } from '../button'
+import { ButtonGroup } from '../buttonGroup'
+import { Disclaimer } from '../disclaimer'
+import { Paragraph } from '../paragraph'
+import { SubHeader } from '../subHeader'
+import { StyledSpecialOffer, StyledSpecialOfferPicture } from './SpecialOffer.style'
+
+type Picture = Readonly<{
+  url: string
+  alt: string
+}>
+
+type Link = Readonly<{
+  label: string
+  url: string
+}>
+
+type Button = Readonly<{
+  label: string
+  onClick: () => void
+}>
+
+type CTALinks = [Link] | [Link, Link]
+type CTAButtons = [Button] | [Button, Button]
+
+type SpecialOfferBaseProps = Readonly<{
+  picture: Picture
+  title: string
+  description: string
+  informationLink?: Link
+}>
+
+export type SpecialOfferLinksProps = Readonly<
+  SpecialOfferBaseProps & {
+    ctaLinks: CTALinks
+  }
+>
+export type SpecialOfferButtonsProps = Readonly<
+  SpecialOfferBaseProps & {
+    ctaButtons: CTAButtons
+  }
+>
+
+function SpecialOffer(props: SpecialOfferLinksProps): JSX.Element
+function SpecialOffer(props: SpecialOfferButtonsProps): JSX.Element
+function SpecialOffer({
+  picture,
+  title,
+  description,
+  informationLink,
+  ctaLinks,
+  ctaButtons,
+}: SpecialOfferBaseProps &
+  Readonly<{
+    ctaLinks?: CTALinks
+    ctaButtons?: CTAButtons
+  }>) {
+  return (
+    <StyledSpecialOffer>
+      <StyledSpecialOfferPicture src={picture.url} alt={picture.alt} />
+      <SubHeader>{title}</SubHeader>
+      <Paragraph>{description}</Paragraph>
+      {informationLink && (
+        <Disclaimer useInfoIcon>
+          <Button status={ButtonStatus.UNSTYLED} href={informationLink.url}>
+            {informationLink.label}
+          </Button>
+        </Disclaimer>
+      )}
+      {ctaLinks && (
```

Comment:
you could have a function to resolve that outside the render method

Reply:
I'm not sure of seeing what you mean here. Do you think of the function called by the render method that, giving the full props, will returns links or buttons?

Reply:
I think it's not necessary anymore with the `actions` renaming there is only one case to handle